tv_apps/smart-home/js/edit.js

```diff
@@ -231,8 +246,40 @@
     },
 
     deleteCard: function(scrollable, nodeElem) {
-      this._concealPanel(scrollable, nodeElem);
-      this.cardManager.removeCard(parseInt(nodeElem.dataset.idx, 10));
+      this.isNavigable = false;
+      document.activeElement.blur();
+      this.modalDialog.open(
+        'Are you sure you want to delete this card?',
+        [
+          {
+            text: 'cancel',
```

Comment:
This field should be textId or l10nId instead of text. Please rename this field. If possible, please keep the field text and use l10n as the second priority. Like:
```
if (text) {
  dom.textContent = text;
} else {
  dom.setAttribute('....', l10nId || 'ok');
}

```